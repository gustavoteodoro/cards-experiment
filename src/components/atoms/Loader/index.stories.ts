import Loader from './index.vue';

export default { title: 'Atoms/Loader' };

export const Default = () => ({
  components: { Loader },
  template: '<Loader />'
});
