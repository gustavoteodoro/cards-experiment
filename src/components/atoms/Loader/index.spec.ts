import { mount } from "@vue/test-utils";
import Loader from "./index.vue"

describe("Loader", () => {
  it('renders correctly', () => {
    const wrapper = mount(Loader)
    expect(wrapper.element).toMatchSnapshot()
  })
})
