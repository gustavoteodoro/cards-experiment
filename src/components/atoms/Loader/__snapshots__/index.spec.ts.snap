// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<div
  class="lds-ring"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
